configure.ac

```diff
@@ -475,7 +475,6 @@ AC_CACHE_CHECK([for strdup(s)],
 char *t = "Test";
 char *s = strdup(t);
 if (!s || !(s[0]=='T'&&s[1]=='e'&&s[2]=='s'&&s[3]=='t'&&s[4]=='\0') || s == t) res = 1;
-if (s) free (s);
```

Comment:
This looks odd -- IIRC when this test program was PoCed, at least in some build systems it warned about leaks so `free()` was added. What did your instrumentation complain about - lack of header for it? Maybe better fix it from that vector :)

Reply:
Sorry for the delay - it would be OK to add another commit. I'm currently not at a computer so can't check quickly, but hope the `HAVE_STDLIB_H` is discovered in the `configure.ac` (or the m4's it calls) before this check. If not, feel free to move it until after that test is done, so you can safely `ifdef ... include` it :)

Thanks!

Reply:
(and then maybe `ifdef ... free()` to be pedantic?..)